content/webapp/services/prismic/fetch/index.ts

```diff
@@ -62,12 +66,19 @@ export function createClient(
  * it felt silly not to abstract
  */
 
-type Params = Parameters<
+export type GetByTypeParams = Parameters<
   GetServerSidePropsPrismicClient['client']['getByType']
 >[1];
 
+/** Returns true if a document has one of the content types specified in `contentType`. */
+function hasMatchingContentType(document: PrismicDocument, contentType: ContentType | ContentType[]): boolean {
+  return isString(contentType)
+    ? document.type === contentType
+    : contentType.map(ct => ct as string).indexOf(document.type) !== -1;
```

Comment:
does the `map` here do anything other than keep TypeScript happy?

Reply:
Just keeping TypeScript happy.

Reply:
Would definitely rather use `includes` here rather than `indexOf`, as it states intent much more clearly - we've had it in JS since 2016 and I very rarely see a use for `indexOf` now.

Reply:
Have fixed locally, will push when I sort out this compiler issue.

Reply:
Also, isn't the `map` unnecessary if it's just for TS - wouldn't `(contentType as string[]).includes(document.type)` work?

Reply:
Also fixed.